test/report.js

```diff
@@ -30,6 +30,15 @@ const patientTwo = {
     type: "patient"
 }
 
+const admin = {

```

Comment:
I'm confused why we need an admin for the create report interaction

Reply:
Creating reports doesn't need an admin, but getting a reports can only be done by an admin.